docs/eigenlayer/operator-guides/operator-sets.md

```diff
@@ -0,0 +1,370 @@
+
+##  Registration
+
+The registration flow follows the pattern illustrated below and is completed in one or two transactions depending on the requirements set by the AVS:
+
+![](/img/operator-guides/operator-registration-allocation.png)
+
+
+
+
+##  Unique Stake Allocation & Deallocation
+
+Note: Please review the entire [Unique Stake Allocation & Deallocation ELIP-002](https://github.com/eigenfoundation/ELIPs/blob/main/ELIPs/ELIP-002.md) in detail. The following is not a complete description of the Slashing and Operator Sets upgrade and is qualified in its entirety by reference to the [Unique Stake Allocation & Deallocation ELIP-002](https://github.com/eigenfoundation/ELIPs/blob/main/ELIPs/ELIP-002.md#unique-stake-allocation--deallocation).
+
+
+
+Unique Stake is a mechanism to guarantee slashable stake can only be allocated to one Operator Set at a time. Unique Stake benefits both Operators and AVSs with certain properties when reasoning about EigenLayer slashing: 
+
+* **Greater control over slashing risk:** With Unique Stake, the risk of slashing is isolated to the individual AVS and Operator Set, and Operators can control how much of their stake any AVS can slash. AVSs are not exposed to risk from any other AVS or their slashings.  
+* **Guaranteed slashable stake:** AVSs have the tools they need to understand the amount of Unique Stake that can be slashed at a given time across their Operator Sets.   
+* **Permissionless onboarding of AVSs:** Since slashing is localized to individual AVSs, there is no need for a common veto committee which means launching an AVS on EigenLayer remains permissionless.
+
+When Stakers deposit assets on EigenLayer, they are stored in accounting contracts known as Strategies. Strategies are different expressions of security on EigenLayer. Today, they represent different types of restaked assets (e.g., tokens) delegated to Operators that AVSs can leverage for securing their services and upholding cryptoeconomic guarantees. In order to make delegations slashable, Operators must allocate individual proportions of them as Unique Stake to Operator Sets. Allocations are exclusively slashable by the AVS that created that Operator Set.
+
+In figure 3 below, Operator 1 has a delegation of 100 staked ETH for the ETH Strategy. Operator 1 then allocates proportions of that ETH as Unique Stake in Operator Sets across several AVSs. 
+
+
+
+![Operator Allocations to Operator Sets](/img/operator-guides/operator-sets-figure-3.png)  
+***Figure 3: Operator Allocations to Operator Sets***
```

Comment:
Let's remove the numbers

Reply:
When you say "let's remove the numbers" ..  Which specific line numbers in the file would you like for me to remove? There are a lot of numbers, examples, and related diagrams in this section.